site/index.qmd

```diff
@@ -180,231 +201,150 @@ a:hover {
 
 ::::
 
-:::
-
-<!-- GET STARTED SECTION -->
-
-::: {.column-screen}
-
-:::: {.flex .flex-wrap .justify-around .get-started}
-
-::: {.w-10-ns}
-<!-- FILLER COLUMN DO NOT REMOVE -->
-:::
+<!-- CENTERED WHITE BOX WITH FIXED WIDTH & FULL HEIGHT -->
+::: {.center .br3 .shadow-4 .bg-white .pa5 .w-80-ns}
 
+<!-- TOP ROW: Setup + Admin | Model Dev | Model Validation -->
+:::: {.flex .flex-wrap .justify-around .mb5}
 
-::: {.w-25-ns .mb4}
-
-:::: {.ba .b--black-10 .br3 .shadow-4 .bg-white .pt4 .pb3 .h-100 .grow .tc}
-
-:::: {.ph4}
-
-:::: {.navy .f4 .tl}
-[for administrators]{.smallcaps}
+::: {.w-25-ns .mb3}
+:::: {.items-start}
+![](/homepage_docsite/setup_icon.svg){.mb3 width="48"}
 ::::
+### Setup + Administration
+<span class="light-text">
```

Comment:
Global comment: Please aim to use Quarto Markdown and relevant CSS classes wherever possible. We typically use raw HTML only as a fallback. Between Quarto Markdown and Tachyons CSS, this should happen only rarely.